arch/x86/crc32_pclmulqdq_tpl.h

```diff
@@ -196,7 +195,7 @@ static void partial_fold(const size_t len, __m128i *xmm_crc0, __m128i *xmm_crc1,
     __m128i xmm_a0_0, xmm_a0_1;
     __m128 ps_crc3, psa0_0, psa0_1, ps_res;
 
-    xmm_shl = _mm_load_si128((__m128i *)pshufb_shf_table + (len - 1));
+    xmm_shl = _mm_load_si128((__m128i *)(pshufb_shf_table + 4*(len - 1)));
```

Comment:
Add spaces and parans for readability? Up to you..
```c
xmm_shl = _mm_load_si128((__m128i *)(pshufb_shf_table + (4 * (len - 1))));
```

Reply:
Applied.